sar_pipeline/aws/preparation/burst_utils.py

```diff
@@ -107,72 +131,114 @@ def get_burst_ids_and_start_times_for_scene_from_asf(
+                burst_info[burst_id]["pols"].append(pol)
+
+    if len(burst_info) == 0:
         raise FileExistsError(
             "No burst id's for scene could be found on the ASF. Ensure input values are correct "
             "or set `--scene_data_source CDSE` as recent scenes may not be available on ASF, or the scene is missing"
         )
 
-    return burst_ids, burst_sts
+    return burst_info
 
 
-def make_static_layer_base_url(
-    static_layers_s3_bucket: str,
-    static_layers_collection: str,
-    static_layers_s3_project_folder: str,
-    s3_region: str = "ap-southeast-2",
-) -> str:
-    """Make the base url to the static layers from the paths provided
+def get_burst_info_for_scene_from_cdse(
+    scene: str, burst_prefix: str = "t", lowercase: bool = True
+) -> dict:
+    """Get the burst ids, start-times, polarisations and geometries for a scene from CDSE
 
     Parameters
     ----------
-    static_layers_s3_bucket : str
-        Bucket containing static layer
-    static_layers_collection : str
-        collection static layers belong to
-    static_layers_s3_project_folder : str
-        project folder within bucket if exists
-    s3_region : str, optional
-        aws region code, by default "ap-southeast-2"
+    scene : str
+        the scene id. e.g. S1A_IW_SLC__1SSH_20220101T124744_20220101T124814_041267_04E7A2_1DAD
+    burst_prefix : str
+        A prefix to add to the burst string. default 't'. For example:
+        070_149822_IW3 -> t070_149822_IW3 with burst_prefix = 't'.
+    lowercase: bool
+        convert the burst to lowercase. default to True to match workflow output.
+
+    Raises
+    -------
+    FileExistsError:
+        The scene does not exist on the ASF.
 
     Returns
     -------
-    str
-        The url to the index file where static layers are stored for user
-        visibility
+    dict
+        unique dict of scene burst ids mapped to start_times and geometries
+        {'t070_149822_IW3' : {start_time : datetime.datetime, geometry: shapely.geometry}}
     """
-    root_static_layer_path = make_rtc_s1_static_s3_subpath(
-        s3_project_folder=static_layers_s3_project_folder,
-        collection=static_layers_collection,
-        burst_id="",
-    )
-    return (
-        f"https://{static_layers_s3_bucket}.s3.{s3_region}.amazonaws.com"
-        f"/index.html?prefix={root_static_layer_path}"
-    )
+
+    base_url = "https://catalogue.dataspace.copernicus.eu/odata/v1/Bursts"
+    query = f"$filter=ParentProductName eq '{scene}.SAFE'&$top=100"
+    url = f"{base_url}?{query}"
+
+    response = requests.get(url)
+    response.raise_for_status()  # Raise error for bad responses
+
+    results = response.json().get("value", [])
+    burst_info = {}
+
+    for b in results:
+        track_number = int(b.get("RelativeOrbitNumber"))
+        esa_burst_id = int(b.get("BurstId"))
+        subswath = b.get("SwathIdentifier")
+        # construct the unique JPL burst id
```

Comment:
Suggest providing a link to JPL documentation (if any) that demonstrates that this is the structure of their burst id. Should help in future if things change.